BigriverBookstore-apple/AppDelegate.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppDelegate.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Cocoa
+import Spine
+
+@NSApplicationMain
+class AppDelegate: NSObject, NSApplicationDelegate {
+
+    let baseURLpath = "http://jsonapiplayground.reyesoft.com"
```

Comment:
@c-Viorel I think we should move the constant in a separated file, something like NetworkingConstants. The main reason is to keep AppDelegate clean and also to extend easier the baseURL per environment if the case.